README.md

```diff
@@ -10,11 +10,11 @@
 
 This repository contains all the Request Network developer tools written in TypeScript.
 
-Join the [Request Hub][requesthub-slack-url] to get in touch with us.
+Join the [Request Discord][requesthub-discord-url] to get in touch with us.
 
 [website-url]: https://request.network
 [whitepaper-url]: https://request.network/assets/pdf/request_whitepaper.pdf
-[requesthub-slack-url]: https://request-slack.herokuapp.com/
+[requesthub-discord-url]: https://request.network/discord/
```

Comment:
Please change this to `request-discord-url` as well.

Reply:
Fixed [b455df7](https://github.com/RequestNetwork/requestNetwork/pull/1032/commits/b455df7fc1989a2e6fcd5d3f361c26f2e3d9f155)